generators/cloudfoundry/index.js

```diff
@@ -46,38 +46,38 @@ module.exports = CloudFoundryGenerator.extend({
             message: 'Name to deploy as:',
             default: this.baseName
         },
-        {
-            type: 'list',
-            name: 'cloudfoundryProfile',
-            message: 'Which profile would you like to use?',
-            choices: [
-                {
-                    value: 'dev',
-                    name: 'dev'
-                },
-                {
-                    value: 'prod',
-                    name: 'prod'
-                }
-            ],
-            default: 0
-        },
-        {
-            when: function(response) {
-                return databaseType != 'no';
+            {
```

Comment:
I double checked - the IntelliJ Idea IDE uses the .editorconfig from our repository to format the code like you saw in the PR.

Could you please try and see if your IDE (I'm hoping you use Idea too) does anything differently ?